frontend/components/ProductHeader/components/Rating/components/RatingCount/index.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
```

Comment:
I investigated this and it's happening because the "_BEFORE" hack is too slow. Header is rendering before the BEFORE portal and it will always appear for some time.

Also, I think it won't really work in a stable manner since it uses a different stream to decide if to hide original Header component.

I'd say we should do it in another way: register into HEADER portal and there conditionally render original header or just Rating stars.

Since our Portals have no "abstain" feature, we might want to just adjust the HEADER portal so it passed the `Header` component as prop and then the extension can decide what to render.

Also (for your consideration). I don't like copying components from themes, maybe there's a way to not copy the `Render` component.

Reply:
I think that the described issue is related to the asynchronously loaded script for the `item` route. The extension bundle loads a bit after the page bundle. From the 2nd time it doesn't happen anymore.
I don't like the idea to pass the `Header` component to the portal props, since it would be the only one within a bunch of portals within the page component. I'd like to check how it behaves within a live environment.